.github/workflows/publish-to-bcr.yaml

```diff
@@ -0,0 +1,40 @@
+# Publishes to the Bazel Central Registry.
+#
+# Based on .github/workflows/publish-to-bcr.yml from bazel-contrib/rules_scala v7.0.0.
+# See .bcr/README.md.
+name: Publish to the Bazel Central Registry
+
+on:
+  # Run from release.yml.
+  workflow_call:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+    secrets:
+      bcr_publish_token:
+        required: true
+
+  # In case of problems, enable manual dispatch from the GitHub UI.
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        required: true
+        type: string
+
+jobs:
+  publish-to-bcr:
+    uses: bazel-contrib/publish-to-bcr/.github/workflows/publish.yaml@v0.1.0
+    with:
+      tag_name: ${{ inputs.tag_name }}
+      # bazelbuild/bazel-central-registry fork used to open a pull request.
+      registry_fork: EngFlow/bazel-central-registry
+      # No need to create a draft since PRs are opened by a machine account
+      draft: false
```

Comment:
Will this run from a machine account?

Reply:
Discussed internally: Yes, an account with no access to private repos. The `repo` scope grants a token full access to all repos the account may access, so we want to limit damage if the token leaks.